src/wasm/wasm-ir-builder.cpp

```diff
@@ -546,7 +557,14 @@ Result<> IRBuilder::makeBreak(Index label) {
 
 // Result<> IRBuilder::makeSwitch() {}
 
-// Result<> IRBuilder::makeCall() {}
+Result<> IRBuilder::makeCall(Name func, bool isReturn) {
```

Comment:
Should we add a comment documenting that makeCall is a departure from the `wasm-builder.h` `makeCall`, in that there is an assumption the Function already exists before creating the call to it.

Reply:
Can do 👍